test/client/fixtures/sandbox/event/unload-test.js

```diff
@@ -30,6 +30,19 @@ asyncTest('BEFORE_UNLOAD_EVENT must be called last (GH-400)', function () {
         });
 });
 
+test('hammerhead UNLOAD_EVENT must be called', function () {
```

Comment:
We don't use a `must` work in test names (from recent time ).
So, please, rename your test to `UNLOAD_EVENT should be called`.

Also, we have a small piece of unstable tests in hammerhead and testcafe.
You PR don't affect tests.
To restart tests just add a comment `@testcafe-build-bot  \retest`